deploy/00_setup-dev.ts

```diff
@@ -10,11 +10,13 @@ const func: DeployFunction = async function ({
 
   const { deployer, manager, guardian, owner } = await getNamedAccounts();
 
+  await forkNetwork(chain, forkBlock[chain]);
```

Comment:
This should be fine, but for my sake, can you describe the exact problem that this solves?
Testing-running local deployment on a forked network?

Reply:
The motivation is that on the MultichainPostman the constructor calls the executor to get the address of the Multichain executor. We would never be capable of deploying this locally (without a fork) since the _anyCall contract does not exist.
![Screenshot 2022-10-10 at 09 38 00](https://user-images.githubusercontent.com/91997054/194827574-bf8e8d3d-db1b-4657-8284-fb06c03730cb.png)